.github/changelog.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+m_branch=m;
+changelog_file=CHANGELOG.md;
+
+# fetch master since we might be in a shallow clone
+git fetch origin "$m_branch:$m_branch" --depth=1
+
+changed=0;
+for log in */"$changelog_file"; do
```

Comment:
Changelog is in the root of this repo, not a subdirectory, so the leading `*` shouldn't be here

Reply:
Hmm, I can remove the star, but I would prefer to have one script for all projects

Reply:
If I do `for log in "$changelog_file" */"$changelog_file"; do`, this should work, right?

Reply:
Ehh, I will need to update all repos... This is fun

Reply:
You don't have to, we can leave them as is as it already works

Reply:
Yes, but then the next time I need the latest script, I'll have hard time remembering which repo it was with the latest version

Reply:
And actually, one of the repos with changelog in the root was using the script that works with changelogs in `*/`. So it's a good thing to keep everything in sync.